### ISSUE TITLE
V5.8.0.2 - 32Bit Install vcruntime140.dll error

### ISSUE THREAD

**Issue by Jonathan-Garber** [multiOTP/multiOTPCredentialProvider#32]
When installing 5.8.0.2 on a 32bit system with the 32-bit VC++ 2015-2019 Redist. The included DLL in PHP Folder returns an error when trying to run PHP.

```
vcruntime140.dll is either not designed to run on Windows or it contains an error.
```
Is this not 32-bit compatible?

**Comment by multiOTP** [comment]
Hello Jonathan,
Which 32-bit operating system are you using ?
Regards,
Andre

**Comment by Jonathan-Garber** [comment]
Windows 7 32 bit
During install the installer installed the included vc redist package but
failed to ever activate the provider due to the dll error with the included
vcruntime140.dll in the PHP folder.

So I went ahead and took a copy of the vcruntime140.dll from system32 and
overwrote the installer provided dll. Since the bundled copy of PHP is
32bit this copied dll works fine.

I've had no further errors during testing of the included PHP or multiotp.

**Comment by multiOTP** [comment]
Ok, thanks for the feedback.
vcruntime140.dll has nothing to do in the PHP folder, it was a building error.
This has been fixed for the next release (5.8.0.3).
Thanks for your appreciated feedback.
regards,
Andre

**Comment by Jonathan-Garber** [comment]
Yes I kind of wondered why the install had it and didn't just use the
system one after the vc redist installed.

So I just copied the known good one into it lol.
